concordium-contracts-common/src/types.rs

```diff
@@ -1205,11 +1256,18 @@ impl From<Vec<u8>> for OwnedParameter {
 }
 
 impl OwnedParameter {
+    /// Get [`Self`] as the borrowed variant [`Parameter`].
     pub fn as_parameter(&self) -> Parameter { Parameter(self.0.as_ref()) }
 
-    /// Construct an `OwnedParameter` by serializing the input using its
+    /// Construct an [`Self`]` by serializing the input using its
     /// `Serial` instance.
     pub fn new<D: Serial>(data: &D) -> Self { Self(to_bytes(data)) }
+
+    /// Construct an [`Self`] from a vector of bytes.
+    pub fn from_bytes(bytes: Vec<u8>) -> Self { Self(bytes) }
+
```

Comment:
I don't like this since we have a "from_bytes" method that deals with parsing/deserializing values. So that will cause confusion.

Moreover we would ideally be named with `_unchecked` since not every vector is a valid parameter.